docs/userguide.md

```diff
@@ -271,12 +271,12 @@ kubectl patch clusterrole kubefed-role --type='json' -p='[{"op": "add", "path":
         }
 }]'
 ```
-This example is for cluster scoped federation deployment. For namespaced federation deployment,
-you can patch role `kubefed-role` in the kubefed namespace instead.
+This example is for cluster scoped KubeFed deployment. For namespaced KubeFed deployment,
+you can patch role `kubefed-role` in the `kubefed` namespace instead.
```

Comment:
`kubefed` namespace implies that it is the name of the namespace.  Namespace-scoped kubefed can be deployed to any namespace, so 'KubeFed system namespace' is preferable to indicating a specific namespace.

Reply:
Updated, thanks!